aocpy/templates/test_solution.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python
+import pytest
+import solution
+
+@pytest.mark.parametrize('data,expect',
+    [('sample_input_1', 'sample_output_1'),
+     ('sample_input_2', 'sample_output_2')]
+)
+def test_part1(data, expect):
+    assert solution.part_1(data) == expect
+
+@pytest.mark.parametrize('data,expect', 
+    [('sample_input_1', 'sample_output_1'),
+     ('sample_input_2', 'sample_output_2')]
+)
+def test_part2(data, expect)
```

Comment:
Tiny nitpick - could this please include the underscore from the function names? e.g. `test_part_1`

Reply:
Done